assets/helpers/page/page.js

```diff
@@ -36,6 +36,12 @@ import storeReferrer from 'helpers/tracking/awin';
 
 import { type Action } from './pageActions';
 
+// TODO: ideally the `stage` will be injected via config
```

Comment:
`webpack.prod.js` is already defining `process.env.NODE_ENV='production'` in PROD. We could use this already or add it to `webpack.dev.js`

Reply:
👍 Great shout! Done!